qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/customnavview/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -88,71 +84,21 @@ import com.mapbox.navigation.qa_test_app.dump.NavigationViewApiDumpInterceptor
 import com.mapbox.navigation.qa_test_app.view.base.DrawerActivity
 import com.mapbox.navigation.ui.base.lifecycle.UIBinder
 import com.mapbox.navigation.ui.base.lifecycle.UIComponent
-import com.mapbox.navigation.ui.maneuver.model.ManeuverExitOptions
-import com.mapbox.navigation.ui.maneuver.model.ManeuverPrimaryOptions
-import com.mapbox.navigation.ui.maneuver.model.ManeuverSecondaryOptions
-import com.mapbox.navigation.ui.maneuver.model.ManeuverSubOptions
-import com.mapbox.navigation.ui.maneuver.model.ManeuverViewOptions
 import com.mapbox.navigation.ui.maps.NavigationStyles
-import com.mapbox.navigation.ui.maps.route.RouteLayerConstants
-import com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions
-import com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions
-import com.mapbox.navigation.ui.maps.route.line.model.RouteLineColorResources
-import com.mapbox.navigation.ui.maps.route.line.model.RouteLineResources
 import com.mapbox.navigation.utils.internal.toPoint
 
 @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
 class MapboxNavigationViewCustomizedActivity : DrawerActivity() {
```

Comment:
NOTE: I've restructured this activity to make it easier for us to add new customization options.
I've prefixed all switch/spinner handler methods with `toggleXXXX and grouped them with code folding regions.
<img width="300" src="https://user-images.githubusercontent.com/2678039/196790038-66d23f4f-2c12-4013-a544-230b8436e9e3.png" />